### ISSUE TITLE
Updating to 10.0.8 did not worked

### ISSUE THREAD

**Issue by JMendezH** [owncloud/core#31583]
### Steps to reproduce
1. Updating to 10.0.8
2. Start
3. The update was unsuccessful. 

### Expected behaviour
Open de User/ Password request

### Actual behaviour
Can't login

### Server configuration
**Operating system**:

**Web server:**

**Database:**

**PHP version:**

**ownCloud version:** (see ownCloud admin page)

**Updated from an older ownCloud or fresh install:**

**Where did you install ownCloud from:**

**Signing status (ownCloud 9.0 and above):**

```
Login as admin user into your ownCloud and access 
http://example.com/index.php/settings/integrity/failed 
paste the results into https://gist.github.com/ and puth the link here.
```

**The content of config/config.php:**

```
Log in to the web-UI with an administrator account and click on
'admin' -> 'Generate Config Report' -> 'Download ownCloud config report'
This report includes the config.php settings, the list of activated apps
and other details in a well sanitized form.

or 

If you have access to your command line run e.g.:
sudo -u www-data php occ config:list system
from within your ownCloud installation folder

*ATTENTION:* Do not post your config.php file in public as is. Please use one of the above
methods whenever possible. Both, the generated reports from the web-ui and from occ config:list
consistently remove sensitive data. You still may want to review the report before sending.
If done manually then it is critical for your own privacy to dilligently
remove *all* host names, passwords, usernames, salts and other credentials before posting.
You should assume that attackers find such information and will use them against your systems.
```

**List of activated apps:**

```
If you have access to your command line run e.g.:
sudo -u www-data php occ app:list
from within your ownCloud installation folder.
```

**Are you using external storage, if yes which one:** local/smb/sftp/...

**Are you using encryption:** yes/no

**Are you using an external user-backend, if yes which one:** LDAP/ActiveDirectory/Webdav/...

#### LDAP configuration (delete this part if not used)

```
With access to your command line run e.g.:
sudo -u www-data php occ ldap:show-config
from within your ownCloud installation folder

Without access to your command line download the data/owncloud.db to your local
computer or access your SQL server remotely and run the select query:
SELECT * FROM `oc_appconfig` WHERE `appid` = 'user_ldap';


Eventually replace sensitive data as the name/IP-address of your LDAP server or groups.
```

### Client configuration
**Browser:**

**Operating system:**

#### Web server error log
```
Insert your webserver log here
```

#### ownCloud log (data/owncloud.log)
```
Insert your ownCloud log here
```

#### Browser log
```
Insert your browser log here, this could for example include:

a) The javascript console log
b) The network log 
c) ...
```

**Comment by ownclouders** [comment]
[GitMate.io](https://gitmate.io) thinks possibly related issues are https://github.com/owncloud/core/issues/31424 (Update to 10.0.8 was unsuccessful - column "min_value" does not exist), https://github.com/owncloud/core/issues/31361 (Wrong displayname after update (10.0.3 -> 10.0.8)), https://github.com/owncloud/core/issues/14088 (Updating to 8.0: Could not automatically fix legacy storage), https://github.com/owncloud/core/issues/31300 (update to 10.0.8 failed, the server not accessible), and https://github.com/owncloud/core/issues/16777 (Files and several other apps aren't working after update to 8.0.4).

**Comment by JMendezH** [comment]
Hello, these are the errors:


Repair warning: Exception
Repair warning: Archives of type application/octet-stream are not supported
Repair warning: You have incompatible or missing apps enabled that could not be found or updated via the marketplace.
Repair warning: Please install or update the following apps manually or disable them with: occ app:disable templateeditor
Repair warning: For manually updating, see https://doc.owncloud.org/server/10.0/go.php?to=admin-marketplace-apps


Preparing update
Set log level to debug
Turned on maintenance mode
Repair warning: Exception
Repair warning: Archives of type application/octet-stream are not supported
Repair warning: You have incompatible or missing apps enabled that could not be found or updated via the marketplace.
Repair warning: Please install or update the following apps manually or disable them with: occ app:disable templateeditor
Repair warning: For manually updating, see https://doc.owncloud.org/server/10.0/go.php?to=admin-marketplace-apps
OC\RepairException: Upgrade is not possible


________________________________
De: ownClouders <notifications@github.com>
Enviado: martes, 29 de mayo de 2018 13:07
Para: owncloud/core
Cc: JMendezH; Author
Asunto: Re: [owncloud/core] Updating to 10.0.8 did not worked (#31583)


GitMate.io<https://gitmate.io> thinks possibly related issues are #31424<https://github.com/owncloud/core/issues/31424> (Update to 10.0.8 was unsuccessful - column "min_value" does not exist), #31361<https://github.com/owncloud/core/issues/31361> (Wrong displayname after update (10.0.3 -> 10.0.8)), #14088<https://github.com/owncloud/core/issues/14088> (Updating to 8.0: Could not automatically fix legacy storage), #31300<https://github.com/owncloud/core/issues/31300> (update to 10.0.8 failed, the server not accessible), and #16777<https://github.com/owncloud/core/issues/16777> (Files and several other apps aren't working after update to 8.0.4).

**Comment by patrickjahns** [comment]
This is not a bug -  please see this thread in our community for further assistance;
https://central.owncloud.org/t/update-from-10-0-7-to-10-0-8-failed/13635

**Comment by JMendezH** [comment]
Did you noticed I already have that version, it's not possible to upgrade to the same one, it shouldn't been requesting the upgrade.


________________________________
De: Juan José Méndez Hernández <jmendezh@live.com>
Enviado: miércoles, 30 de mayo de 2018 16:58
Para: owncloud/core
Asunto: Re: [owncloud/core] Updating to 10.0.8 did not worked (#31583)


Hello, these are the errors:


Repair warning: Exception
Repair warning: Archives of type application/octet-stream are not supported
Repair warning: You have incompatible or missing apps enabled that could not be found or updated via the marketplace.
Repair warning: Please install or update the following apps manually or disable them with: occ app:disable templateeditor
Repair warning: For manually updating, see https://doc.owncloud.org/server/10.0/go.php?to=admin-marketplace-apps


Preparing update
Set log level to debug
Turned on maintenance mode
Repair warning: Exception
Repair warning: Archives of type application/octet-stream are not supported
Repair warning: You have incompatible or missing apps enabled that could not be found or updated via the marketplace.
Repair warning: Please install or update the following apps manually or disable them with: occ app:disable templateeditor
Repair warning: For manually updating, see https://doc.owncloud.org/server/10.0/go.php?to=admin-marketplace-apps
OC\RepairException: Upgrade is not possible


________________________________
De: ownClouders <notifications@github.com>
Enviado: martes, 29 de mayo de 2018 13:07
Para: owncloud/core
Cc: JMendezH; Author
Asunto: Re: [owncloud/core] Updating to 10.0.8 did not worked (#31583)


GitMate.io<https://gitmate.io> thinks possibly related issues are #31424<https://github.com/owncloud/core/issues/31424> (Update to 10.0.8 was unsuccessful - column "min_value" does not exist), #31361<https://github.com/owncloud/core/issues/31361> (Wrong displayname after update (10.0.3 -> 10.0.8)), #14088<https://github.com/owncloud/core/issues/14088> (Updating to 8.0: Could not automatically fix legacy storage), #31300<https://github.com/owncloud/core/issues/31300> (update to 10.0.8 failed, the server not accessible), and #16777<https://github.com/owncloud/core/issues/16777> (Files and several other apps aren't working after update to 8.0.4).
